rust/gui-client/src-tauri/src/client/about.rs

```diff
@@ -19,6 +19,7 @@ mod tests {
         let git = super::get_git_version();
 
         assert!(cargo != "Unknown", "{}", cargo);
+        assert!(cargo.starts_with('1'));
```

Comment:
This is a little bit over-fitted but it prevents `cargo-mutants` from passing "xyzzy" as a version

Reply:
This is testing for an env var set by cargo, could we assume that this works w/o it?

Reply:
Yeah the test is redundant but it brings the mutant number down. In this PR I was going for easy ones instead of useful ones